django_migrations_ci/management/commands/migrateci.py

```diff
@@ -71,8 +71,14 @@ def handle(
         elif parallel is not None:
             parallel = int(parallel)
 
-        storage_class = get_storage_class(storage_class)
-        default_storage_class = get_storage_class(
+        if hasattr(settings, "STORAGES") and "default" in settings.STORAGES:
```

Comment:
This `settings` is not Django settings.

https://github.com/businho/django-migrations-ci/blob/main/django_migrations_ci/conf.py#L22